README.md

```diff
@@ -29,6 +29,12 @@ Change directory back to root of repository
 cd ..
 ```
 
+## Installation
+```
+1. Install python dependencies with pip 
+
+```
```

Comment:
```suggestion

1. Install python dependencies with pip 
```

Reply:
@matthew-brett No sorry, I was just testing the instructors' tag as one of the instructor told me to.